src/main/content/_assets/js/javadocs.js

```diff
@@ -306,6 +359,9 @@ function addClickListener(contents) {
             window.history.pushState(state, null, hashParams);
         }
     })
+    contents.find("a").on("contextmenu", function(event) {
+        console.log("event", event);
+    })
```

Comment:
This was meant for a different issue right?

Reply:
oops, I still have the codes there to investigate the right click. Thanks for catching it.